mlflow/server/js/.eslintrc.js

```diff
@@ -471,9 +466,13 @@ module.exports = {
         'plugin:jsx-a11y/recommended',
         'plugin:@typescript-eslint/recommended',
       ],
-      plugins: ['prettier', 'react', '@typescript-eslint', '@emotion'],
+      plugins: ['prettier', '@typescript-eslint', '@emotion'],
       parser: '@typescript-eslint/parser',
 
+      // BEGIN-EDGE
+      // While we're waiting to have code sharing here, the following rules
+      // are plucked from js/packages/config-eslint/presets/createConfig.js
+      // END-EDGE
```

Comment:
Shouldn't this be exported?

Reply:
hmm those blocks seem to be left in [`.eslintrc.oss.js`](https://src.dev.databricks.com/databricks/universe/-/blob/mlflow/web/js/oss/.eslintrc.oss.js) file which is copied in to export package after changes in https://github.com/databricks/universe/pull/400945
I'll just remove those blocks, it seem to work

Reply:
thanks for catching this! 🙏